infra.bs

```diff
@@ -715,6 +715,11 @@ performing a set of steps on each <a for=list>item</a> in order, use phrasing of
 "<a for=list>For each</a> |item| of <var ignore>list</var>", and then operate on |item| in the
 subsequent prose.
 
+<p>To <dfn export for=list,stack,queue,set>clone</dfn> a <a>list</a> |list| is to create a new
```

Comment:
I think we should expand on this in two ways:

> NOTE: this is a "shallow clone", as the items themselves are not cloned in any way

plus something about how cloning a specific type of list (stack/queue/ordered set) produces a list which is also one of those things. Maybe with an example?